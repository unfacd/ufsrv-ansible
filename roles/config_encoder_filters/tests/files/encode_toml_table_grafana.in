verbose_logging: false

servers:
  - host: 127.0.0.1
    port: 389
    use_ssl: no
    start_tls: no
    ssl_skip_verify: no
    bind_dn: cn=admin,dc=grafana,dc=org
    bind_password: grafana
    search_filter: (cn=%s)
    search_base_dns:
      - dc=grafana,dc=org
  - attributes:
      name: givenName
      surname: sn
      username: cn
      member_of: memberOf
      email: email
  - group_mappings:
      - group_dn: cn=admins,dc=grafana,dc=org
        org_role: Admin
      - group_dn: cn=users,dc=grafana,dc=org
        org_role: Editor
      - group_dn: "*"
        org_role: Viewer
