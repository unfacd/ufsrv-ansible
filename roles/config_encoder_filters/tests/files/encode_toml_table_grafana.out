verbose_logging = false

[[servers]]
bind_dn = "cn=admin,dc=grafana,dc=org"
bind_password = "grafana"
host = "127.0.0.1"
port = 389
search_base_dns = ["dc=grafana,dc=org"]
search_filter = "(cn=%s)"
ssl_skip_verify = false
start_tls = false
use_ssl = false

[servers.attributes]
email = "email"
member_of = "memberOf"
name = "givenName"
surname = "sn"
username = "cn"

[[servers.group_mappings]]
group_dn = "cn=admins,dc=grafana,dc=org"
org_role = "Admin"

[[servers.group_mappings]]
group_dn = "cn=users,dc=grafana,dc=org"
org_role = "Editor"

[[servers.group_mappings]]
group_dn = "*"
org_role = "Viewer"
